### DISCUSSION TITLE
Does the technical preview has a time/usage gap?

### DISCUSSION THREAD

**Discussion by Oyshoboy** [community/community#6690] [Copilot Conversations] [answered]
Just curious:
1. Is there any time limits while on technical preview?
2. Is it okay to share my GitHub account info with my friend to try Copilot out?

**Comment by linktlh** [comment, marked as answer]
1. Not that I'm aware of

2. No, that breaks ToS B.3
> Your login may only be used by one person — i.e., a single login may not be shared by multiple people

**Reply by Oyshoboy** [reply]
Thanks for answer!
Btw, do you know where I can grab a ToS?

**Reply by Oyshoboy** [reply]
> Thanks for answer! Btw, do you know where I can grab a ToS?

Nvm, found it:
https://docs.github.com/en/github/site-policy/github-terms-of-service
